lib/EditableList/EditableListForm.js

```diff
@@ -374,6 +375,19 @@ class EditableListForm extends React.Component {
     return visibleFields;
   }
 
+  getColumnMapping() {
+    const {
+      editable,
+      columnMapping,
+    } = this.props;
+
+    if (editable) {
+      columnMapping.actions = <FormattedMessage id="stripes-smart-components.actions" />;
+    }
+
+    return columnMapping;
```

Comment:
pedantic/bikeshedding: I generally like to treat `props` as immutable (I know `props.columnMapping` is an object reference, and we're not changing that ref, so it technically isn't being mutated, but still...). Strictly adhering to that, you could do something like: 
```
const map = {...columnMapping};
if (editable) {
  map.actions = <FormattedMessage id="stripes-smart-components.actions" />;
}

return map;
```

Reply:
Thanks, @zburke, corrected.